handlers/templates/pages/guest-link-index.html

```diff
@@ -82,6 +86,50 @@
           });
       });
     });
+
+    document
+      .querySelectorAll('[aria-label="Enable"], [aria-label="Disable"]')
+      .forEach((toggleBtn) => {
+        toggleBtn.addEventListener("click", () => {
+          const id = toggleBtn.getAttribute("pico-link-id");
+          // switch enable/disable
+          if (toggleBtn.getAttribute("aria-label") === "Enable") {
```

Comment:
The `if-else` in the loop body means that the Enable and Disable cases share basically no code. I think we should either:

1. Use [classList.toggle](https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/toggle) to share code outside the if-else
2. Split it up into two separate loops with no `if-else`.